### PR TITLE
Add notes for clustering

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
- Add note for multi-region considerations

### CODE REVIEW

**Review by sumerjabri — APPROVED** [review]

File: source/system-administrators/activities/clustering/cluster-troubleshooting.rst [hunk 1]
```diff
@@ -17,11 +17,17 @@ Whenever your cluster has a Git or DB sync failure, the following logs may appea
...
 
 .. _cluster-troubleshooting-db-sync-fail-log:
 
-<add sample log of failed DB sync>
+.. code-block:: text
+   :caption: *Sample log for a cluster DB sync failure*
+
+   <add sample log of failed DB sync>
 
 An email will also be sent to configured recipients to inform them of the failure.
```
Comment (sumerjabri) [inline source/system-administrators/activities/clustering/cluster-troubleshooting.rst:32]: ... to the configured list of recipients ...

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: source/new-ia/by-role/system-admin/clustering/cluster-troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+   :width: 35%
+
+|
+
+After reviewing the node states (logs and the database), the fix can be any valid intervention on the database.
+After fixing the database, the node may then be restarted.
+
+If an admin reviews the node states and thinks everything is fine but still receives DB sync errors, the admin
+may decide if MariaDB should ignore those errors and continue. To ignore the errors, a manual intervention is
+required and may be done by following the instructions `here <https://mariadb.com/kb/en/set-global-sql_slave_skip_counter/>`__
```
Comment (jmendeza) [inline source/new-ia/by-role/system-admin/clustering/cluster-troubleshooting.rst:90]: If there is a startup failure, Studio will stop and MariaDB will be down. Can we add here instructions to start it up in order to perform the manual fix?

**Review by jmendeza — CHANGES_REQUESTED** [review]
Please review my comment

**Review by jmendeza — APPROVED** [review]

File: source/new-ia/by-role/system-admin/clustering/cluster-troubleshooting.rst [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+========================================
+Cluster Troubleshooting |enterpriseOnly|
+========================================
+
+Whenever your cluster has a Git or DB sync failure, the following logs may appear:
+
+.. _newIa-cluster-troubleshooting-git-sync-fail-log:
+
+.. code-block:: text
+   :caption: *Sample log for a cluster Git sync startup failure*
```
Comment (sumerjabri) [inline source/new-ia/by-role/system-admin/clustering/cluster-troubleshooting.rst:18]: ... an authoring cluster ...

File: source/system-administrators/activities/clustering/cluster-troubleshooting.rst [hunk 4]
```diff
@@ -0,0 +1,188 @@
+:is-up-to-date: True
```
Comment (sumerjabri) [inline source/system-administrators/activities/clustering/cluster-troubleshooting.rst:1]: Same for this file.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**